client/src/components/NavBar/index.js

```diff
@@ -64,20 +67,27 @@ const DropdownMenu = ({ history }) => (
 const StyledDropdown = styled(Dropdown)`
   cursor: pointer;
 
-  > svg {
+  svg {
     height: 18px;
     width: 18px;
   }
 `;
 
-const Wrapper = styled.div`
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  padding: 1rem 4.5rem;
-  background: whitesmoke;
+const Wrapper = styled(Header)`
+    position: fixed;
```

Comment:
Added a fixed NavBar - we may need to update the layouts to account for it, and give them some extra padding on top.

Reply:
👍🏼